src/cloud/live/site-launch-checklist.md

```diff
@@ -16,51 +16,80 @@ Before you deploy to the Production environment, download the [Launch checklist]
 
-Another option for apex domain is to add A records, which maps a domain name to the Fastly IP addresses:
+   On Starter projects, you must add the domains to your project. See [Manage domains]({{ site.baseurl }}/cloud/cdn/configure-fastly-customize-cache.html#manage-domains).
 
--  `151.101.1.124`
--  `151.101.65.124`
--  `151.101.129.124`
--  `151.101.193.124`
+-  SSL/TLS certificate provisioned for your production environments.
 
-### TLS and Fastly
+   If you added the ACME challenge records for your Production domains during the Fastly setup process, Magento has already validated the domains and provisioned the certificate.  The certificate will be applied to your Production environment automatically when you update the DNS configuration to point your Production domains to the Fastly service. If you did not pre-provision the certificate, or if you updated your domains, Magento must complete domain validation and provision the certificate, which takes up to 12 hours.
 
-Magento provides a Let's Encrypt SSL/TLS certificate for each environment. This certificate is required for Fastly to serve secure traffic over HTTPS.
+{:.procedure}
+To update DNS configuration for site launch:
+
+1. Update the following DNS configuration for your Production site:
+
+   -  Set all necessary redirects, especially if you are migrating from an existing site
+   -  Set the zone’s root resource record to address the hostname
+   -  Lower the value for the Time-to-Live (TTL) to refresh DNS information to point customers to the correct Production store
+
+      We recommend a significantly lower TTL value when switching the DNS record. This value tells the DNS how long to cache the DNS record. When shortened, it refreshes the DNS faster. For example, you can change the TTL value from 3 days to 10 minutes when you are testing your site. Be advised that shortening the TTL value adds load to the web server.
+
+1. Add CNAME records to point the top-level domains and subdomains for your Production environment to the Fastly service `prod.magentocloud.map.fastly.net`, for example:
+
+   | Domain or Subdomain  | CNAME
+   |----------------------|------
+   | www.your-domain.com  | prod.magentocloud.map.fastly.net
+   | mystore.your-domain.com | prod.magentocloud.map.fastly.net
+
+1. If needed, add A records to map the apex domain (`your-domain.com`) to the following Fastly IP addresses:
+   | Apex domain     | ANAME
+   |-----------------|--------
+   | your-domain.com | `151.101.1.124`
+   | your-domain.com | `151.101.65.124`
+   | your-domain.com | `151.101.129.124`
+   | your-domain.com | `151.101.193.124`
+
+1. Update the Magento Base URL.
+
+   -  Use SSH to log in to the Production environment.
+
+      ```bash
+      magento-cloud ssh
+      ```
+
+   -  Use the Magento CLI to change the base url for your store
+
+      ```
+      php bin/magento setup:store-config:set --base-url="https://mcstaging.your-domain.com/"
```

Comment:
Feedback from @cgarrigues:

Step 4 should have the final domain as the --base-url argument, not staging:

php bin/magento setup:store-config:set --base-url="https://www.your-domain.com/"


```suggestion
      php bin/magento setup:store-config:set --base-url="https://www.your-domain.com/"
```